### PR TITLE
Migrate to pytest-selenium

### PR DESCRIPTION
**Pull request by davehunt**
I picked up on the initial work by @justinpotts to migrate to the new version of our pytest plugin.

Pinging @m8ttyB and @johngian for review.

Adhoc jobs:
* [Sauce Labs](http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/mozillians.adhoc.pytest-selenium/4/)
* [Grid (Windows)](http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/mozillians.adhoc.pytest-selenium/6/)
* [Grid (OSX)](http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/mozillians.adhoc.pytest-selenium/8/)

### CODE REVIEW

File: pages/base.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def login(self, email, password):
         from browserid import BrowserID
         pop_up = BrowserID(self.selenium, self.timeout)
         pop_up.sign_in(email, password)
-        WebDriverWait(self.selenium, 20).until(lambda s: self.is_user_loggedin)
+        WebDriverWait(self.selenium, self.timeout).until(lambda s: self.is_user_loggedin)
```
Comment (m8ttyB) [inline pages/base.py]: <3

File: pages/base.py [hunk 2]
```diff
@@ -64,22 +64,22 @@ def create_new_user(self, email, password):
 
         WebDriverWait(self.selenium, 20).until(lambda s: self.is_user_loggedin)
```
Comment (m8ttyB) [inline pages/base.py]: Should this also be changed to `WebDriverWait(self.selenium, self.timeout).until(lambda s: self.is_user_loggedin)`?

File: pages/page.py [hunk 3]
```diff
@@ -56,13 +36,13 @@ def get_url_current_page(self):
...
         try:
-            self._selenium_root.find_element(*locator)
+            self.selenium.find_element(*locator)
             return True
         except NoSuchElementException:
             return False
         finally:
             # set back to where you once belonged
-            self.selenium.implicitly_wait(self.testsetup.default_implicit_wait)
+            self.selenium.implicitly_wait(10)
```
Comment (m8ttyB) [inline pages/page.py]: More of a curiosity than a nit, why hard code the implicit waits?

File: tests/conftest.py [hunk 4]
```diff
@@ -6,6 +6,19 @@
 import requests
 
 
+@pytest.fixture(scope='session')
+def capabilities(capabilities):
+    capabilities.setdefault('tags', []).append('mozillians')
+    return capabilities
+
+
```
Comment (m8ttyB) [inline tests/conftest.py]: Stylistically, are 2 line breaks needed?

File: pages/base.py (hunk 2 above)
Comment (davehunt) [inline pages/base.py]: Thanks, I spotted a couple more, too.

File: pages/page.py (hunk 3 above)
Comment (davehunt) [inline pages/page.py]: This isn't a change in value, it's just pytest-selenium doesn't set a default implicit wait.

File: tests/conftest.py (hunk 4 above)
Comment (davehunt) [inline tests/conftest.py]: Yes, it's PEP8. http://legacy.python.org/dev/peps/pep-0008/#blank-lines

Comment (m8ttyB) [inline tests/conftest.py]: Cool beans :)

**Comment by m8ttyB** [comment]
r+

lgtm

File: pages/page.py [hunk 5]
```diff
@@ -115,15 +97,9 @@ def return_to_previous_page(self):
     def get_relative_path(self, url):
         self.selenium.get(u'%s%s' % (self.base_url, url))
 
-    def find_element(self, *locator):
```
Comment (bebef1987) [inline pages/page.py]: I know this was merged but just to understand

Why did we removed this methods?

Comment (davehunt) [inline pages/page.py]: They do not align with our style guide. Page objects can simply call the `find_element*` methods directly.

**Pull request merged**